scheduler/launchd.go

```diff
@@ -0,0 +1,149 @@
+package scheduler
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+
+	homedir "github.com/mitchellh/go-homedir"
+)
+
+type launchd struct{}
+
+// Enable creates the launchd script and then loads it using launchctl
+func (ld *launchd) Enable() (launchdWasLoaded bool, err error) {
```

Comment:
As before, if the purpose of `Enable` is just to instantiate `fileFunctionsPassthrough{}` so we can run tests against `enable`, then that's all it should do.

In this case, the functionality seems to be 50/50 split between `Enable` and `enable` !